docs/language/stdlib/subprocessing.rst

```diff
@@ -16,3 +16,31 @@ Functions
    Run a command *cmd*. It is parsed to get a list of
    commands and arguments. If something (while parsing or running),
    a run error will be shown.
+
+.. py:function: run_python_code(cmd)

```

Comment:
Hmm... it seems like this block is being... ignored? See [the source](https://diddiparser2--8.org.readthedocs.build/en/8/_sources/language/stdlib/subprocessing.rst.txt) and [the results](https://diddiparser2--8.org.readthedocs.build/en/8/language/stdlib/subprocessing.html).